components/script/dom/element.rs

```diff
@@ -1230,7 +1230,9 @@ impl Element {
         if !self.is_connected() {
             return false
         }
-        return true
+        // allow fullscreen
```

Comment:
I don't think this comment adds anything. However, let's add a link to https://fullscreen.spec.whatwg.org/#fullscreen-element-ready-check for this method.